src/main/java/org/fcrepo/importexport/ArgParser.java

```diff
@@ -189,13 +206,33 @@ protected Config parseConfiguration(final String[] args) {
             }
         }
 
+        validateConfig(config);
+
         // Write command line options to disk
         saveConfig(config);
 
         return config;
     }
 
     /**
+     * Validation routines for checking interdependent configuration parameters.
+     * @param config
+     */
+    private void validateConfig(final Config config) {
+        // verify bagit config
+        if (config.getBagProfile() != null &&
+                config.getBagConfigPath() == null) {
+            throw new RuntimeException("A bagit config path must be set when using a bagit profile.");
```

Comment:
See I didn't realize that because up till now we haven't been enforcing those restrictions so I thought this was adding an additional layer of complexity. I still think if the profile doesn't require extra tags we shouldn't require a person to make a config file. But I also don't like the idea of another option to turn this off. So I'll change my vote and say leave it as is.